test/verify/machineslib.py

```diff
@@ -1479,7 +1489,8 @@ def installWithErrorTest(dialog):
         # Fake the osinfo-db data in order that it will allow spawn the installation - of course we don't expect it to succeed -
         # we just need to check that the VM was spawned
         fedora_28_xml = self.machine.execute("cat /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml")
-        self.machine.execute("cat /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml > /tmp/fedora-28.xml")
+        self.machine.execute("cp /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml /tmp/fedora-28.xml")
+        self.addCleanup(self.machine.execute, "mv /tmp/fedora-28.xml /usr/share/osinfo/os/fedoraproject.org/fedora-28.xml")
```

Comment:
Overwriting /usr is brittle and error -prone, and you couldn't do that on fedora-coreos. Seems better to get into the habit of bind-mounting it instead of `cp`ing.